### PR TITLE
Add Docker file for easier development

### PR DESCRIPTION
**Pull request by zarubaf**
Docker can simplify setting up the development environment significantly. A next PR will focus on updating the README to make use of this docker file.

### CODE REVIEW

**Comment by zarubaf** [comment]
@MikeOpenHWGroup can you take a look? Afaik all other committers are on holidays.

File: util/docker/README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+docker build . -f util/docker/Dockerfile.ubuntu -t openhwgroup/cva6:latest
+```
+
+To run the image
+
+```
+docker run -it -v `pwd`:/repo --env="DISPLAY=docker.for.mac.host.internal:0" openhwgroup/cva6
+```
+
+This will open a prompt where you `cd /repo` and execute the `make` commands.
```
Comment (cathales) [inline util/docker/README.md:16]: Could we use `WORKDIR /repo` ?

**Review by cathales — COMMENTED** [review]
Hello Florian,

Here are a few suggestions

**Comment by JeanRochCoulon** [comment]
Florian, I see that @cathales have suggested some modifications. What is the status ?  On my side, I approuve the PR and can merge. Tell me up !

**Comment by JeanRochCoulon** [comment]
Hello @zarubaf and @cathales 
THis PR is open since 2 months. This feature could be interesting to execute in a clean room environmnent. But as discussion is down since several weeks, I propose to close it. Do not hesitate to resuscitate him if necessary.

**Pull request closed without merging**